event/operation.go

```diff
@@ -1,145 +0,0 @@
-package event
```

Comment:
the code in this file was copied as-is to `types.go` to better match the rest of the packages, and to prevent having to write custom case for just this service.